microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -966,33 +982,37 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 129.
-* This checkbox is present in Microsoft Edge Stable 128.
+* This checkbox is present in Microsoft Edge Canary 131.
+* This checkbox is present in Microsoft Edge Stable 129.
 
 
 <!-- ====================================================================== -->
-## Performance panel: enable sidebar
-<!-- checkbox has no "(?)" link pointing to this anchor wording -->
-
-_new as of Microsoft Edge 128_
+## Performance panel: enable performance insights
```

Comment:
need desc for "Performance panel: enable performance insights"
@captainbrosset 

https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3283#performance-panel-enable-performance-insights